app/factory.py

```diff
@@ -34,6 +35,8 @@ def create_app(app_name=a_name, **kwargs):
         initialize(kwargs.get('celery'), app)
     # allow swagger to actually work when deployed
     app.wsgi_app = ProxyFix(app.wsgi_app, x_proto=1, x_host=1)
+    # Setup the Flask-JWT-Extended extension
+    jwt = JWTManager(app)
```

Comment:
leave out the return value here. I specifically do this (in methods like `initialize`) to send the clear message that we do not do shit like `from app import jwt`. i.e it should just be:

    JWTManager(app)